server/routerlicious/kubernetes/nginx/values-ppe.yaml

```diff
@@ -11,6 +11,18 @@ controller:
     default-ssl-certificate: default/wu2-ppe-tls-certificate
   admissionWebhooks:
     enabled: False # Disable admission webhooks endpoint to keep moving parts to a minimum as long as we don't use it
+  service:
+    annotations:
+      # This annotation is necessary for external traffic to reach the pods correctly in AKS.
```

Comment:
Nit: This appears to be the first time "AKS" is mentioned here. Can we define it?

Reply:
(And in the other template)

Reply:
Done!